engine/Editor/SceneEditor.cpp

```diff
@@ -1,5 +1,8 @@
 #include "SceneEditor.h"
 
+irrklang::ISound* audio;
```

Comment:
irrkalng specific code should not be in any other file. try encapsulating this datatype or finding another way to use it.

Reply:
No worries, that was just there while I was testing the dynamic audio. Has been removed.